jobs/ghslacker.groovy

```diff
@@ -1,10 +0,0 @@
-import util.Plumber
-
-def p = new Plumber(name: 'sqre/ci-ci/ghslacker', dsl: this)
```

Comment:
This has been replaced by SQuaRE's checkerboard service that Build Engineering is not responsible for.

Removal is correct.